.github/workflows/build-for-release.yml

```diff
@@ -12,39 +13,45 @@ jobs:
         os: [ubuntu-latest, windows-latest]
         include:
           - os: ubuntu-latest
-            exe_suffix: ""
             target: x86_64-unknown-linux-musl
+            archive: tar.gz
           - os: windows-latest
-            exe_suffix: ".exe"
             target: x86_64-pc-windows-msvc
+            archive: zip
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: stable
           target: ${{ matrix.target }}
           override: true
       - name: Cache dependencies
-        uses: Swatinem/rust-cache@v1
-      - uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --release --locked --verbose --target ${{ matrix.target }}
+        uses: Swatinem/rust-cache@v2
+      - name: Build
+        run: cargo build --release --locked --verbose --target ${{ matrix.target }}
+      - name: Pack for Linux
+        if: matrix.os == 'ubuntu-latest'
+        env:
+          RELEASE_NAME: dotter-${{ matrix.target }}
+        run: |
+          mkdir $RELEASE_NAME
+          cp -r target/${{ matrix.target }}/release/dotter -t $RELEASE_NAME
+          cp -r completions -t $RELEASE_NAME
+          tar -zcvf $RELEASE_NAME.${{ matrix.archive }} -C $RELEASE_NAME .
+      - name: Pack for Windows
```

Comment:
I think it's better to upload 3 assets: unix binary, windows binary, and completions archive. This makes life better for users who download only the binaries.

Hopefully this doesn't make much of a difference for the PKGBUILD side?

Reply:
The decompression of `tar.gz` or `zip` isn't difficult at all for anybody. This convenience as you mention does not hold true. (Excuse me, I may be too harsh)

Change PKGBUILD won't affect users.

Reply:
> > I set the workflow running event to push tags. Now we can use git push --tags in command line to start the workflow.
> 
> What happens to the uploaded assets if a tag is created without a release?

Did you mean you dislike releasing on tag created?

Once the tag is created, the action will definitely start and create new assets.

Reply:
> That's the way I use the project and the change makes it less convenient for me, so I won't approve it.

Well, you are the BOSS. I will rewrite it after all things get determined.